libevm/ExtVMFace.h

```diff
@@ -169,6 +169,33 @@ class EnvInfo
     u256 m_gasUsed;
 };
 
+/// Represents a call result.
+///
+/// @todo: Replace with evmc_result in future.
+struct CallResult
+{
+    evmc_status_code status = EVMC_INTERNAL_ERROR;
```

Comment:
For alignment reasons yhis struct might be more compact if you put `enum` member after `owning_bytes_ref` member

Reply:
Logically, the status should be first, because the rest of fields depend on value here.

Reply:
I swapped address with output. It matches the order in evmc_result. Other suggestions also applied.